x-pack/legacy/plugins/siem/public/components/drag_and_drop/draggable_wrapper.tsx

```diff
@@ -175,7 +175,7 @@ const DraggableWrapperComponent = React.memo<Props>(
       return () => {
         unRegisterProvider!({ id: dataProvider.id });
       };
-    }, []);
```

Comment:
@spalger what in case we want to treat `useEffect` as `componentDidMount` should we use `eslint-disable-line` for that?

Reply:
I suppose so